client/.env-example

```diff
@@ -4,7 +4,7 @@ DB_HOST=<database host>
 DB_USER=<database username, for authentication>
 DB_PASS=<database password, for authentication>
 
-REACT_APP_GOOGLE_MAPS_API_KEY=<Enter API KEY here>
+REACT_APP_GOOGLE_MAPS_API_KEY=AIzaSyBA2RlpJOlVVkN9u7HfZRcKEgfG2Nxc35U
```

Comment:
When it's placed inside the .env-example, like here, it's public. So anyone can use the key now. It should be placed in .env, which has been .gitignored.

Reply:
That was me trying to clear that env error. I'll remove it as soon as I'm home, around noon.

Reply:
Done